src/transformStateWithClones.js

```diff
@@ -7,7 +7,43 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copy = {};
+  const arr = [];
```

Comment:
use proper names for copy and result
https://github.com/mate-academy/js_stateful-clones/blob/master/checklist.md